README.md

```diff
@@ -5,4 +5,51 @@ This includes all helper & installer methods.
 
 It might not be up to that state yet, but that's the goal for this library. If
 you see anything puppet-specific that you'd like to pull into this library out
-of beaker, please do, we would love any help that you'd like to provide. 
\ No newline at end of file
+of beaker, please do, we would love any help that you'd like to provide. 
+
+# How Do I Use This?
+
+### Right Now?
+
+At this point, beaker-puppet is included in beaker, so you don't have to _do_
+anything to get the methods in this library.
+
+You can use these methods in a test by referencing them by name without
+qualifications, as they're included in the beaker DSL by default.
+
+### In beaker's Next Major Version?
+
+In beaker's next major version, the requirement for beaker-puppet will be pulled
+from that repo. When that happens, then the usage pattern will change. In order
+to use this then, you'll need to include beaker-puppet as a dependency right
+next to beaker itself.
+
+Once you've done that & installed the games, in your test, you'll have to
```

Comment:
Are we opinionated about if we want people to do this in their test files or in their helper somewhere? This might be a good place to endorse the usage of helpers to make the DSL available to the entirety of a test run, instead of individual tests.

Also, I think that you mean "installed the gems" not "installed the games". 👾  🎲  🎮